common-content/en/activities/tell-me-the-target/_index.md

```diff
@@ -0,0 +1,10 @@
++++
+title="Tell me the target"
+description="You’ll have to state what the target outcome/output is when we run the code"
```

Comment:
Hey, do we need this "you will have to" preamble?

Reply:
Good catch - this is a hangover from getting messy ideas into a Google doc. I'll remove the preamble and make it all crisp and active